test/Microsoft.Health.Fhir.Shared.Tests.Crucible/CrucibleDataSource.cs

```diff
@@ -88,5 +90,10 @@ private async Task<ServerTestRun> GetTestRunAsync()
 
             return new ServerTestRun(crucible.ServerBase, result);
         }
+
+        public static DataStore GetDataStore()
+        {
+            return _dataStore;
+        }
```

Comment:
This looks like it should be a property

Reply:
Having DataStore as a property, leads to create all other methods as non static and [memberData] should be static so it results in a compilation error